javatests/arcs/android/e2e/testapp/AndroidManifest.xml

```diff
@@ -68,6 +68,13 @@
       android:exported="false"
       android:process=":test"/>
 
+    <!-- Note: We currently enforce running the StorageService on the default process to
+         prevent a potential race in DB creation. The WorkManager tasks will always run in the
+         main process, and they manipulate the DB directly. -->
+    <service
+      android:name="arcs.sdk.android.storage.service.StorageService"
```

Comment:
Would the service declaration conflict with the existing StorageService's AndroidManifest?

Reply:
It replaces it for this application. So the StorageService runs in the default (main) process in TestApplication with this change, rather than a 2nd application process named `:storage`.

Reply:
I understand the intention, just curious why don't we need something like `tools:replace` :)

Reply:
Oh I see what you're saying.

I had some logging to verify that this was actually starting the service on the main process, but I was not reading very well; it's not actually doing what I thought :-D

Thanks for calling this out!